napalm/base.py

```diff
@@ -13,6 +13,12 @@
 # the License.
 import napalm.exceptions
 import sys
+import os
+
+from napalm.exceptions import DriverTemplateNotImplemented, TemplateNotImplemented, TemplateRenderException
+
+from jinja2 import Environment, FileSystemLoader
```

Comment:
Can you import jinja2 instead? Why? To avoid local names collisions. I know code gets longer and we could get into an endless debate here about the usage of `from x import y` vs `import x` but I find way more readable:

```
try:
    do_something()
except jinja2.exceptions.Environment as e:
   do_something_else()
```

than:

```
try:
    do_something()
except Environment as e:
   do_something_else()
```

as in the second case I have to go to the imports to check who might be triggering that `Environment` exception. Another benefit is that we might need to catch later on an exception also called named `Environment` coming from another library (`Environment` is a rather generic name).